AutomatedTesting/Gem/PythonTests/scripting/ScriptEvent_AddRemoveParameter_ActionsSuccessful.py

```diff
@@ -35,50 +22,49 @@ class ScriptEvent_AddRemoveParameter_ActionsSuccessful:
      5) Remove Parameter from Event
 
     Note:
-     - This test file must be called from the Open 3D Engine Editor command terminal
      - Any passed and failed tests are written to the Editor.log file.
         Parsing the file or running a log_monitor are required to observe the test results.
 
     :return: None
     """
 
-    def __init__(self):
-        self.editor_main_window = None
-        self.asset_editor = None
-        self.asset_editor_widget = None
-        self.asset_editor_row_container = None
-        self.asset_editor_menu_bar = None
+    # Preconditions
+    import os
+    from PySide2 import QtWidgets
+    from editor_python_test_tools.utils import Report
+    import pyside_utils
+    import scripting_utils.scripting_tools as tools
+    import azlmbr.legacy.general as general
+    from editor_python_test_tools.QtPy.QtPyO3DEEditor import QtPyO3DEEditor
+    from scripting_utils.scripting_constants import (ASSET_EDITOR_UI)
 
-    @pyside_utils.wrap_async
-    async def run_test(self):
+    general.idle_enable(True)
+    general.close_pane(ASSET_EDITOR_UI)
 
-        # Preconditions
-        general.idle_enable(True)
-        general.close_pane(ASSET_EDITOR_UI)
+    # 1) Get a handle on our O3DE QtPy objects then initialize the Asset Editor object
+    qtpy_o3de_editor = QtPyO3DEEditor()
+    # Close and reopen Asset Editor to ensure we don't have any existing assets open
+    qtpy_o3de_editor.close_asset_editor()
+    qtpy_asset_editor = qtpy_o3de_editor.open_asset_editor()
 
-        # 1) Open the asset editor
-        result = tools.open_asset_editor()
-        Report.result(Tests.asset_editor_opened, result)
+    # 2) Create new Script Event Asset
+    qtpy_asset_editor.click_menu_bar_option("Script Events")
```

Comment:
Add to scripting/asset editor consts.

Reply:
imported constant and replaced the hard coded string